cranelift/codegen/meta/src/shared/instructions.rs

```diff
@@ -1615,6 +1615,13 @@ pub(crate) fn define(
 
         When this instruction compares integer vectors, it returns a vector of
         lane-wise comparisons.
+
+        When comparing scalars, the result is `1` if the condition holds, and
```

Comment:
Just a formatting preference:

When comparing scalars, the result is:

   - `1` if the condition holds.
   - `0` if the condition does not hold.
    
When comparing vectors, the result is 
   - `-1` (ie all ones) in each lane if the condition holds.
   -  `0` in each lane if the condition does not hold.